tests/connection_e2e_fcr2gcp_test.go

```diff
@@ -0,0 +1,21 @@
+package tests
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCloudRouter2GcpCreateConnection(t *testing.T) {
+	// retryable errors in terraform testing.
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		TerraformDir: ".../examples/fabric/v4/cloudRouterConnectivity/cloudRouter2gcp",
```

Comment:
Sorry to nit, there's 3`.` instead of two here. You've got my approval for the PR though.

Reply:
Good catch! Updated